server/migrations/20230425152008-createTestPlanColumns.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+    async up(queryInterface, Sequelize) {
+        const t = await queryInterface.sequelize.transaction();
+        try {
+            await queryInterface.addColumn(
```

Comment:
The foreign key constraints you've added declared in `models/TestPlan.js`  has to be added to the migration scripts as well